### DISCUSSION TITLE
sjmisc::rec() - val.labels function is not working

### DISCUSSION THREAD

**Discussion by MarcoJCTeng** [quarto-dev/quarto-cli#5533] [Q&A]
Hi,

I just found out that the `val.labels` function is not working anymore in `sjmisc::rec()`. I have tried to input the labels in both English and Chinese but failed in the end. The following image shows the code and the result of my operation process.

<img width="675" alt="Screenshot 2023-05-10 at 10 23 49 PM" src="https://github.com/strengejacke/sjmisc/assets/123462382/64a648ef-982c-4ded-80d3-6652d70a1ebd">

**Comment by mcanouil** [comment]
I am afraid, this has nothing to do with Quarto.
